doc/changes/dev/13408.newfeature.rst

```diff
@@ -0,0 +1,2 @@
+Add support for multi-wavelength NIRS processing to :func:`mne.preprocessing.nirs.beer_lambert_law`, :func:`mne.preprocessing.nirs.scalp_coupling_index`, :mod:`mne.preprocessing.nirs.nirs`, and SNIRF reader :class:`mne.io.snirf._snirf.RawSNIRF`, by :newcontrib:`Tamas Fehervari`.
```

Comment:
That submodule isn't publicly documented so we can't link to it. Better to just link to the user-facing stuff
```suggestion
Add support for multi-wavelength NIRS processing to :func:`mne.preprocessing.nirs.beer_lambert_law`, :func:`mne.preprocessing.nirs.scalp_coupling_index`, and SNIRF reader :class:`mne.io.snirf._snirf.RawSNIRF`, by :newcontrib:`Tamas Fehervari`.
```